.github/workflows/pypi-cd.yml

```diff
@@ -169,13 +169,13 @@ jobs:
                       fi
                       PB_REL="https://github.com/protocolbuffers/protobuf/releases"
                       ARCH=`uname -p`
-                      if [[ $ARCH == 'x86_64' ]]; then 
+                      if [[ $ARCH == 'x86_64' ]]; then
```

Comment:
If all the changes in this file are just minor whitespace changes, then they should probably just be reverted. It's kind of noisy to have these changes in the PR.